xchange-core/src/main/java/org/knowm/xchange/dto/marketdata/Ticker.java

```diff
@@ -55,6 +56,7 @@
    *     provided
    * @param bidSize The instantaneous size at the bid price
    * @param askSize The instantaneous size at the ask price
+   * @param percentageChange 24h percentage change
```

Comment:
the ticker class is uses not only for 24h time spans

its also used for 5 min, 1h, etc. series

because of this I'm against defining this field as a 24h change

the documentation should be more precise and should state:
- which value is compared: last? high? average?
- when this field will be null? probably then the exchange does not provide it and we cannot calculate it (first ticker in a collection)

ps. in the builder I would suggest to add a `previousTicker(ticker)` method which would calculate the average when its possible